### PR TITLE
OSDOCS-2682: Add release note for egress IPs public cloud support

### PR DESCRIPTION
**Pull request by jboxman-rh**
- https://issues.redhat.com/browse/OSDOCS-2682

Add release note entry for egress IP on public clouds.

Preview: [Egress IP address support for clusters installed on public clouds](https://deploy-preview-39805--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-egress-ip-support-public-clouds)

### CODE REVIEW

**Comment by jboxman-rh** [comment]
@alexanderConstantinescu, how does this look? Thanks!

**Comment by jboxman-rh** [comment]
@alexanderConstantinescu, can you check out this RN entry? Thanks!

**Comment by jboxman-rh** [comment]
@huiran0826, can you confirm? Thanks!

**Review by alexanderConstantinescu — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -63,6 +63,22 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-10-networking"]
 === Networking
 
+[id="ocp-4-10-egress-ip-support-public-clouds"]
+=== Egress IP address support for clusters installed on public clouds
+
+As a cluster administrator you can associate one or more egress IP addresses with a namespace. An egress IP address ensures that a consistence source IP address is associated with traffic from a particular namespace leaving the cluster.
```
Comment (bobfuru) [inline release_notes/ocp-4-10-release-notes.adoc:69]: Minor nits/suggestions:
```suggestion
As a cluster administrator, you can associate one or more egress IP addresses with a namespace. An egress IP address ensures that a consistent source IP address is associated with traffic from a particular namespace that is leaving the cluster.
```

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -63,6 +63,22 @@ This release adds improvements related to the following components and concepts.
...
+
+For the OVN-Kubernetes and OpenShift SDN cluster network providers you can configure an egress IP address on the following public cloud providers:
```
Comment (bobfuru) [inline release_notes/ocp-4-10-release-notes.adoc:71]:
```suggestion
For the OVN-Kubernetes and OpenShift SDN cluster network providers, you can configure an egress IP address on the following public cloud providers:
```

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -63,6 +63,22 @@ This release adds improvements related to the following components and concepts.
...
+
+* Amazon Web Services (AWS)
+* Google Cloud Platform (GCP)
+* Microsoft Azure
+
+To learn more refer to the respective documentation for your cluster network provider:
```
Comment (bobfuru) [inline release_notes/ocp-4-10-release-notes.adoc:77]:
```suggestion
To learn more, refer to the respective documentation for your cluster network provider:
```

**Review by bobfuru — COMMENTED** [review]
A few minor nits and otherwise LGTM.

**Comment by jboxman-rh** [comment]
@bobfuru, I was being economical with my commas, but your way is better, thanks!

**Pull request merged**